src/language/__tests__/blockString-fuzz.js

```diff
@@ -66,3 +70,77 @@ describe('printBlockString', () => {
     }
   }).timeout(20000);
 });
+
+describe('isPrintableBlockString', () => {
+  it('correctly tells if random strings are printable block string', () => {
+    // Testing with length >7 is taking exponentially more time. However it is
+    // highly recommended to test with increased limit if you make any change.
+    for (const fuzzStr of genFuzzStrings({
+      allowedChars: ['\n', '\t', ' ', '"', 'a', '\\'],
+      maxLength: 7,
+    })) {
+      if (isPrintableBlockString(fuzzStr)) {
+        const testStr = '"""' + fuzzStr + '"""';
+
+        let testValue;
+        try {
+          testValue = lexValue(testStr);
+        } catch (e) {
+          continue; // skip invalid values
+        }
```

Comment:
You don't need to skip these invalid values this is the purpose of this fuzz test to detect them.